test/functional/tests/initialize/test_clean_reboot.py

```diff
@@ -31,48 +28,65 @@
 def test_load_after_clean_shutdown(reboot_type, cache_mode, filesystem):
     """
     title: Planned system shutdown test.
-    description: Test for data consistency after clean system shutdown.
+    description: |
+        Test for data consistency after clean system shutdown.
     pass_criteria:
-      - DUT should reboot successfully.
+      - DUT reboot successfully.
       - Checksum of file on core device should be the same before and after reboot.
     """
-    with TestRun.step("Prepare CAS device."):
+    mount_point = "/mnt/test"
+
+    with TestRun.step("Prepare cache and core devices"):
         cache_disk = TestRun.disks["cache"]
+        core_dev = TestRun.disks["core"]
+
         cache_disk.create_partitions([Size(1, Unit.GibiByte)])
+
         cache_dev = cache_disk.partitions[0]
-        core_dev = TestRun.disks["core"]
+
+    with TestRun.step("Start cache and add core"):
         cache = casadm.start_cache(cache_dev, cache_mode, force=True)
         core = cache.add_core(core_dev)
+
+    with TestRun.step("Create filesystem on the core device and mount it"):
         core.create_filesystem(filesystem, blocksize=int(Size(1, Unit.Blocks4096)))
         core.mount(mount_point)
 
-    with TestRun.step("Create file on cache and count its checksum."):
-        test_file = File(os.path.join(mount_point, "test_file"))
-        Dd().input("/dev/zero").output(test_file.full_path).block_size(
-            Size(1, Unit.KibiByte)
-        ).count(1024).run()
+    with TestRun.step("Create file on exported object"):
+        test_file = File(posixpath.join(mount_point, "test_file"))
+
+        dd = (
+            Dd()
+            .input("/dev/zero")
+            .output(test_file.full_path)
+            .block_size(Size(1, Unit.KibiByte))
+            .count(1024)
+        )
+        dd.run()
```

Comment:
```suggestion
        (Dd()
            .input("/dev/zero")
            .output(test_file.full_path)
            .block_size(Size(1, Unit.KibiByte))
            .count(1024)
            .run())
```

Reply:
I would lean towards my version. Breaking down the creation of the object into separate steps make it easier to read/debug